images/dnsmasq/Makefile

```diff
@@ -18,44 +18,37 @@ ARCH ?= amd64
 DNSMASQ_VERSION ?= dnsmasq-2.76
 CONTAINER_PREFIX ?= k8s-dns
 
-ALL_ARCH := amd64 arm arm64 ppc64le
-IMAGE := $(CONTAINER_PREFIX)-dnsmasq-$(ARCH)
-COMPILE_IMAGE := gcr.io/google_containers/kube-cross:v1.6.2-2
+ALL_ARCH := amd64 arm arm64 ppc64le s390x
+IMAGENAME := $(CONTAINER_PREFIX)-dnsmasq
+# TODO: Use gcr.io/google_containers/kube-cross:v1.7.4-2
+COMPILE_IMAGE := gcr.io/google_containers/kube-cross:v1.7.4-1
 OUTPUT_DIR := _output/$(ARCH)
+ML_PLATFORMS=linux/amd64,linux/arm,linux/arm64,linux/ppc64le,linux/s390x
 
 ifeq ($(ARCH),amd64)
-	BASEIMAGE ?= alpine:3.4
-	COMPILE_IMAGE := alpine:3.4
+	BASEIMAGE ?= busybox:glibc
```

Comment:
Does this build correctly? I believe we add several packages with apk. Is apk present in busybox:glibc?